modules/sdk-core/src/coins/susd.ts

```diff
@@ -62,7 +62,7 @@ export class Susd extends BaseCoin {
     return {};
   }
 
-  isWalletAddress(params: VerifyAddressOptions): boolean {
+  isWalletAddress(params: VerifyAddressOptions): Promise<boolean> {
```

Comment:
why promise if we're not denoting it as async?

Reply:
so that yarn doesnt yell. I'll add async prefix to the ones i missed though.